src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx

```diff
@@ -76,7 +76,7 @@ interface IState {
     ignoredUserIds: string[];
     waitingUnignored: string[];
     managingInvites: boolean;
-    invitedRoomAmt: number;
+    invitedRooms: Room[];
```

Comment:
Since you're mainly adding and removing elements of this array, I think it could be better to use a `Set`